students/11th/hanteagyu/login.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>westagram login page</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+   <body>
+      <div class="full">
+          <main>
+              <section class="midbox">
+                  <div class="phone">
+                  </div>
+                  <div class="loginContainor">
+                      <div class = "login">
+                          <img class="logo" src="img/logo_text.png" alt="instagram">
+                          <input class="loginInfo" id="loginID" type="text" placeholder=" 전화번호, 사용자 이름 또는 이메일" />
+                          <input class="loginInfo" id="loginPW" type="password" placeholder=" 비밀번호" />
+                          <button class="loginButton" disabled=disabled>로그인</button>
+                          <div class="Or">
+                              <div class="line"></div>
+                              <div class="orText">또는</div>
+                              <div class="line"></div>
+                          </div>
+                          <button class="facebookButton">
+                              <span class="facebookIcon"></span>
+                              <span class="facebook">Facebook으로 로그인</span>
+                          </button>
+                          <a href="?" class="passwordSearch">비밀번호를 잊으셨나요?</a>
+                      </div>
+                      <div class="signUp">
+                          <p class="signUpText">계정이 없으신가요? <span><a href="?" class="sighlink">가입하기</a></span>
+                          </p>
+                      </div>
+                      <div class="download">
+                          <p class="downloadtext">앱을 다운로드 하세요.</p>
+                      <a href="https://itunes.apple.com/app/instagram/id389801252?pt=428156&ct=igweb.loginPage.badge&mt=8&vt=lo"><img class="downloadapp" src="img/app%20store.png"></a>
+                      <a href="https://play.google.com/store/apps/details?id=com.instagram.android&referrer=utm_source%3Dinstagramweb%26utm_campaign%3DloginPage%26ig_mid%3D5B1C4FC0-FCBD-4DD4-B59F-337F033EED86%26utm_content%3Dlo%26utm_medium%3Dbadge"><img class="downloadapp" src="img/google%20play.png"></a>
```

Comment:
```suggestion
                      <a href="https://itunes.apple.com/app/instagram/id389801252?pt=428156&ct=igweb.loginPage.badge&mt=8&vt=lo">
                        <img class="downloadapp" src="img/app%20store.png">
                      </a>
```
요소들의 부모 자식관계가 잘 드러나도록 줄바꿈 해주세요